measures/measures-schema.yaml

```diff
@@ -110,6 +110,14 @@ definitions:
         description: Possible methods for submitting performance data for the measure.
         type: array
         items: { $ref: #/definitions/methods }
+      eligibilityOptions:
+        description: Eligibility options mirror denominator options in measure specifications. Each option comprises a set of codes used to identify eligible instances of the associated measure.
```

Comment:
Cool, thanks so much for that detail. I've updated the descriptions with the QCDR modifier in https://github.com/CMSgov/qpp-measures-data/pull/28/commits/a6e8e6d360cccc2addf13e70426c0c96c54088e3